test/integration/calculators/part_year_profit_tax_credits_calculator_test.rb

```diff
@@ -2,76 +2,76 @@
 
 module SmartAnswer
   class PartYearProfitTaxCreditsCalculatorTest < ActiveSupport::TestCase
-    context 'tax credits finish in 2015/16 tax year for an ongoing business with accounting period aligned to the tax year' do
+    context 'tax credits finish in 2018/19 tax year for an ongoing business with accounting period aligned to the tax year' do
```

Comment:
Do we have to update all the dates in this file? It seems like a lot to change - can we add new tests for 2018/19 only?